pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
```

Comment:
> I would be happy for something that's more descriptive. Maybe one of your epic half-page names is in order :)

Well, there's a reason 80 characters is too short. :)

This seems to be focused specifically on limits.  I think I would have it match your variable name `LimitRequestOverridesConfig`.